package.yaml

```diff
@@ -17,6 +17,7 @@ dependencies:
   - amazonka-core
   - amazonka-kinesis
   - amazonka-s3
+  - aws-lambda-haskell-runtime >= 2.0
```

Comment:
Just noting that this has to be in stable Stackage before this PR can be merged.

Reply:
Yep, working on it

Reply:
As soon as this gets accepted, we will have it on LTS-13.25 https://github.com/commercialhaskell/lts-haskell/issues/236

Reply:
Merged! This is now in LTS-13.27: https://www.stackage.org/lts-13.27/package/aws-lambda-haskell-runtime-2.0.1